supabase/config.toml

```diff
@@ -8,7 +8,7 @@ enabled = true
 port = 54321
 # Schemas to expose in your API. Tables, views and stored procedures in this schema will get API
 # endpoints. `public` is always included.
-schemas = ["public", "graphql_public"]
+schemas = ["public", "graphql_public", "vault"]
```

Comment:
We need to expose the `vault` schema through the Data API to check the shared secret within the edge function for authorization purposes.